qiskit/visualization/utils.py

```diff
@@ -114,10 +114,17 @@ def get_param_str(op, drawer, ndigits=3):
         param_list = [f"{op.params[0]}[{op.unit}]"]
     else:
         param_list = []
+        truncate_params = False
```

Comment:
Do you really need this variable? (I think its there just to break a long conditional.)

Reply:
If I write the whole conditional inside the `if` statement, I get a pylint error that there are too many boolean expressions in the `if` (max permitted were 5). 

If there is consensus that truncation should happen based on the length of the string, then the logic can be simplified and this variable can be removed.